### PR TITLE
Expose Advisory details in API

### PR DESCRIPTION
**Pull request by ykodwani01**
Solves issue https://github.com/nexB/vulnerablecode/issues/1433

Signed-off-by: Yash Kodwani <yashkodwani2003@gmail.com>

### CODE REVIEW

**Comment by ykodwani01** [comment]
@pombredanne @ziadhany  Can you please review and let me know if any changes required

**Comment by ziadhany** [comment]
@ykodwani01 the CI ( code style is failing  )

Don't forget to run `make valid` in the terminal to check the code style before you commit.

File: vulnerabilities/api.py [hunk 1]
```diff
@@ -182,6 +183,14 @@ def to_representation(self, instance):
 
         weaknesses = data.get("weaknesses", [])
         data["weaknesses"] = [weakness for weakness in weaknesses if weakness is not None]
+        alias_queryset = instance.aliases.all()
+        data["advisory"] = []
+        for itr in alias_queryset:
+            advisory_objects = Advisory.objects.filter(aliases=[itr.alias])
+            for i in advisory_objects:
```
Comment (ziadhany) [inline vulnerabilities/api.py:190]: please try to use meaningful variable names when iterating (  `i`,  `itr` )

```suggestion
            for advisory_object in advisory_objects:
```

**Review by ziadhany — CHANGES_REQUESTED** [review]
@ykodwani01 Looks good! but I think the most important data we want to expose ( `date_collected`, `created_by` and `url` )

**Comment by ykodwani01** [comment]
@ziadhany Done with the required changes

**Comment by ziadhany** [comment]
> @ziadhany Done with the required changes.
> 
> There is a problem with DCO, I committed one change directly from github which caused error and now Its failing the test. Can you suggest something to resolve it.

You can read these two articles :
- https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md
- https://medium.com/@michaelyuan_88928/a-complete-guide-to-dco-for-open-source-developers-fa063c17d9e7

**Comment by ykodwani01** [comment]
New PR is raised at https://github.com/nexB/vulnerablecode/pull/1461

**Pull request closed without merging**